docusaurus/docs/tutorials/build-a-logs-data-source-plugin.md

```diff
@@ -360,6 +360,7 @@ import { catchError, lastValueFrom, of, switchMap, Observable } from 'rxjs';
 
 export class ExampleDatasource
   extends DataSourceApi<ExampleQuery, ExampleOptions>
+  // Implement DataSourceWithLogsContextSupport and its methods
```

Comment:
```suggestion
```

I think that this new comment is redundant and we don't use anything similar at other places in this documentaion.